.github/workflows/mirror-certs.yaml

```diff
@@ -22,5 +22,4 @@ jobs:
           author_email: devops@va.gov
           branch: main
           cwd: .
-          add: *.cer
```

Comment:
No longer required as we pass in acceptance flag. See previous [PR](https://github.com/department-of-veterans-affairs/platform-va-ca-certificate/pull/1/files#r866367202) commend for additional details